libmscore/lyricsline.cpp

```diff
@@ -183,7 +183,7 @@ SpannerSegment* LyricsLine::layoutSystem(System* system)
       if (tick() >= stick) {
             layout();
             if (ticks().isZero())                 // only do layout if some time span
-                  return 0;
+                  return nullptr;
```

Comment:
Should this be made part of the [MuseScore coding style](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules)?
It isn't strictly needed, as a 0 in any pointer context automagically turns into a nullptr (C++) or NULL (C), but might be better doe selfdocumenting code?

Reply:
Maybe so.  I know it gets mentioned in code reviews, so I just think of it as part of the style even if it is not officially.

Reply:
OK, now added in https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules#Returning-pointers